framework/resources/Datasources/table_stats/analyze_tables.sql

```diff
@@ -20,4 +20,10 @@ analyze table `table_stats/nested_scalar_cols` compute statistics(glossary.title
 analyze table `table_stats/Tpch0.01/parquet/customer` compute statistics;
 analyze table `table_stats/Tpch0.01/parquet/part` compute statistics;
 analyze table `table_stats/Tpch0.01/parquet/partsupp` compute statistics;
+analyze table  `table_stats/empty` compute statistics;
+
+analyze table `table_stats/pp_some_cols/1991/1` compute statistics(p_name,p_type,p_size);
```

Comment:
Hey @anishareddyv, not clear where the `pp_some_cols` dataset is being added.. can you please clarify?

Reply:
@Agirish yes I had missed committing it in this PR. I have done it now. Thanks!